examples/RunSingleModel.jl

```diff
@@ -66,4 +68,7 @@ print_summary(NC_Model)
 save_summary(NC_Model, output_file_isolated)
 
 # plot Sankey plot of NC model
-plot_sankey(NC_Model)
\ No newline at end of file
+plot_sankey(NC_Model)
+
+# plot business plan of NC model
+#business_plan_plot(NC_Model)
```

Comment:
Each file shall end with an empty line, please add a "{enter}" at the end of the file